validators/changing-node-ip.adoc

```diff
@@ -1,31 +1,21 @@
+
+[IMPORTANT]
+Currently it's not possible to change IP address if your node has any SKALE chains. So in case your node is participating in a chain, the whole process will require more time because foundation need to remove it first.
 
 [NOTE]
 It's recommended to use a **permanent IP that can dynamically point to a node's machine IP (for example, elastic IP)** so you can update machines if needed without having to change the IP recorded in SKALE Manager during node registration.
 
 [IMPORTANT]
-If you must change your node IP in SKALE Manager, you must perform a node exit and re-registration between the time after getBounty is called and before 72 hours (3 days) elapses in the next month. Also, performing a node exit could be expensive to the node operator if it's currently supporting SKALE Chains.
-
-To perform a node exit, and register a new machine and IP:
-
-1. Be sure to backup SKALE Node and SGX Wallet.
-2. Be sure you have your SGX Backup key.
-3. Execute `skale exit start` on the exiting node during the time window between getBounty and 72 hours from the UTC start of the month.
-4. Verify the exit process by executing `skale exit status`. (You don't need to wait for exit to finalize)
-5. Create a new node machine (`skale node init .env`, `skale wallet info`).
-6. From the old machine, move your node address funds to the new node wallet address `skale wallet send [new_address] [amount]`.
-7. Unlink the old node address `sk-val validator unlink-address [OLD_NODE_ADDRESS] `
-8. Link the new node to the validator ID `skale node signature [VALIDATOR_ID]`, `sk-val validator link-address [NEW_NODE_ADDRESS] [NEW_NODE_SIGNATURE]`
-9. Register the node with under the new IP `skale node register --name [NODE_NAME] --ip [NODE_IP] --port [PORT]`.
+If you must change your node IP in SKALE Manager, you must perform the procedure below between the time after getBounty is called and before 72 hours (3 days) elapses in the next month.
 
-To perform a node exit, keep the original node machine and its data and update the IP, follow these steps (NOT RECOMMENDED):
+To change node IP address you need to perform following steps:
 
-1. Perform the general node exit procedure steps 1 and 2 above.
-2. Be sure you have securely saved the sgx_key_name before moving to step 3.
-3. Remove .skale directory, remove all old containers, volumes, etc
-4. Do skale node init again.
-5. Go to `~/.skale/node_data/node_config.json` and replace `sgx_key_name` field with the old one.
-6. Link the node to the validator ID again.
-7. Register node with a new IP.
\ No newline at end of file
+1. Ask core team whether it's currently possible to change IP address.
+2. Backup SKALE Node and SGX Wallet.
+3. Be sure you have your SGX Backup key.
+4. Create new machine.
+5. Restore SGX if needed.
+6. Restore node from backup (`skale node restore`).
```

Comment:
7- send new IP address to core team

Reply:
We already have it by this time. Otherwise we wouldn't be able to execute transaction